assume/common/market_objects.py

```diff
@@ -24,10 +24,14 @@ class Order(TypedDict):
     bid_id: str
     start_time: datetime
     end_time: datetime
-    volume: int  # positive if generation
-    price: int
-    only_hours: OnlyHours | None = None
+    volume: float  # positive if generation
+    price: float
+    accepted_volume: float
     agent_id: str
+    profile: Optional[dict[datetime, float]]  # for block orders
+    accepted_profile: Optional[dict[datetime, float]]  # for block orders
+    bid_type: Optional[str]  # SB, BB, LB
+    only_hours: Optional[OnlyHours]
```

Comment:
Actually, I would like to also split price in "price_ordered" and "price_cleared" for all orders and then for block orders the "price_cleared would also be a dict, since the mcp can change over the time span of the order.

Reply:
yes, makes sense. I have also removed profile and just allow volume to be both a number or a dict